internal/cmd/organization/command_update.go

```diff
@@ -0,0 +1,50 @@
+package organization
+
+import (
+	orgv2 "github.com/confluentinc/ccloud-sdk-go-v2/org/v2"
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/resource"
+)
+
+func (c *command) newUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update the current Confluent Cloud organization.",
+		Args:  cobra.NoArgs,
+		RunE:  c.update,
+	}
+
+	cmd.Flags().String("name", "", "Name of the Confluent Cloud organization.")
+	_ = cmd.MarkFlagRequired("name")
+
+	pcmd.AddOutputFlag(cmd)
```

Comment:
Do we want to add `--context`?
```suggestion
	cmd.Flags().String("name", "", "Name of the Confluent Cloud organization.")
	pcmd.AddOutputFlag(cmd)
	
	_ = cmd.MarkFlagRequired("name")
```